WalletWasabi/Blockchain/Transactions/TransactionStore.cs

```diff
@@ -64,8 +65,9 @@ private void Import(string oldPath, string dbPath, Network network)
 
 			SqliteStorage.BulkInsert(allTransactions);
 
-			Logger.LogInfo($"Removing old '{oldPath}' transaction storage.");
 			File.Delete(oldPath);
+			Logger.LogInfo($"Migration of transaction file '{oldPath}' to SQLite format was finished in {stopwatch.Elapsed} seconds.");
```

Comment:
I would keep the log that the old file was removed.

```suggestion
			Logger.LogInfo($"Old '{oldPath}' transaction storage was removed. Migration to SQLite format was finished in {stopwatch.Elapsed} seconds.");
```